controls/roles/validator-import-lighthouse/tasks/main.yml

```diff
@@ -23,16 +23,17 @@
       }
 
       spawn sh -c { sudo docker run --rm -it \
-                     -e DEBUG_LEVEL=debug \
-                     -e NETWORK=prater \
-                     -e DATADIR="/opt/app/validator" \
-                     -e DIRECTORY="/opt/app/launchpad" \
-                     -v "/opt/app/services/{{ validator_service_name }}/validator":"/opt/app/validator" \
-                     -v "/opt/app/services/{{ validator_service_name }}/launchpad":"/opt/app/launchpad" \
+                     -v $HOME/.lighthouse:/root/.lighthouse \
+                     -v "/opt/app/services/2e83d610-48b8-11ec-802a-330dee8e6b2e/validator":"/opt/app/validator" \
```

Comment:
This is a fixed path, it will break as soon as this actually get's used. However, I don't think we need those CLI imports anymore.

What do you think about deactivating all tests regarding CLI import? It looks like import via API works anyway.

Reply:
CLI import is deactivated